tests/integration/integration_test.py

```diff
@@ -52,7 +52,7 @@ def test_main(self):
     def startup(self):
         self.files_to_cleanup = []
 
-        tmp_cfg_path = self.create_tmp_cfg()
+        tmp_cfg_path = self.create_tmp_cfg(self)
```

Comment:
Why do you pass `self` to a static method?
If there is no special need for accessing the instance, just remove the argument.